lite2/client.go

```diff
@@ -814,6 +831,7 @@ func (c *Client) backwards(
 	now time.Time) error {
 
 	if HeaderExpired(initiallyTrustedHeader, c.trustingPeriod, now) {
+		c.logger.Info("Header Expired")
```

Comment:
Shouldn't this be an error?

Reply:
Yes you're right I did see it before but forgot to change it

Reply:
So are you going to change this or not?